### ISSUE TITLE
Support URL_PREFIX

### ISSUE THREAD

**Issue by fescobar** [fescobar/allure-docker-service#114]
Useful for reverse proxy configuration like NGINX
```
    environment:
      URL_PREFIX: "/reporting"
```

**Comment by fescobar** [comment]
@apulverizer I've almost finished a big refactoring. Please, check your feature in the `beta` version. I hope soon release this together with Security layer feature.

Before pulling, remove any previous `beta` if you have one.

```
  allure:
    image: "frankescobar/allure-docker-service:beta"
```

Please, let me know when you finish and don't use `beta` versions for prod environments, because it will be removed once we finished testing.

Thank you so much again.

**Comment by apulverizer** [comment]
This looks good to me using the `"frankescobar/allure-docker-service:beta"` image. Thanks for cleaning it up and adding this @fescobar

**Comment by fescobar** [comment]
@apulverizer Thank you for your feedback. Give me a few days to finish the security layer and release your code.
Thanks.

**Comment by fescobar** [comment]
@apulverizer I've re-released the version `2.13.5` with your feature. Remove your current version and pull again the `latest`.
https://github.com/fescobar/allure-docker-service/releases/tag/v2.13.5

Also, I've created a new UI: https://github.com/fescobar/allure-docker-service-ui

Thank you so much for your help.

**Comment by apulverizer** [comment]
Looks good, thanks. You can close this.
